app/scrapers/yandex.py

```diff
@@ -0,0 +1,23 @@
+from __future__ import print_function
+from generalized import Scraper
+
+
+class Yandex(Scraper):
+    """Scrapper class for Yandex"""
+
+    def __init__(self):
+        self.url = 'https://www.yandex.com/search'
+        self.defaultStart = 0
+
+    def parseResponse(self, soup):
+        """ Parse the response and return set of urls
+        Returns: urls (list)
+                [[Tile1,url1], [Title2, url2],..]
+        """
+        urls = []
+        for a in soup.findAll('a', {'class': 'link link_theme_normal'}):
+            urls.append({'title': a.getText(), 'link': a.get('href')})
+
```

Comment:
What you have here is correct.  I tend to like list comprehensions but other folks do not.  It is a matter of taste...
```python
        urls = [{'title': a.getText(), 'link': a.get('href')} for a
                in soup.findAll('a', class_='link link_theme_normal')]
```